cloud/aws/modules/secrets_manager/outputs.tf

```diff
@@ -12,3 +12,19 @@ output "database_password" {
   value     = data.aws_secretsmanager_secret_version.postgres_password_version.secret_string
   sensitive = true
 }
+
+output "oidc_client_id" {
+  value     = data.aws_secretsmanager_secret_version.oidc_client_id_version.secret_string
+  sensitive = true
+}
+
+output "oidc_secret" {
+  value     = data.aws_secretsmanager_secret_version.oidc_secret_version.secret_string
+  sensitive = true
+}
+
+output "oidc_discovery_uri" {
```

Comment:
All of these need the prefix applicant ("applicant_oidc_discovery_uri"), since there could be a admin oidc client in the future